### PR TITLE
[feat]: Tasks

### PR DESCRIPTION
**Pull request by RakshithBhat03**
# Tasks

### 1. Task
1.1 Add Task
1.2 Delete Task
1.3 Edit Task
1.4 Mark task as complete

### 2. Modal
2.1 Set task name
2.2 Set task description
2.3 Add time for Pomodoro, short break, and long break
2.4 Data persistence (localStorage).

### CODE REVIEW

**Review by abhij1607 — COMMENTED** [review]
Good folder structure
Neat & clean code
Liked the work with CSS and constants

Good to merge

File: src/components/AddFirstTask/AddFirstTask.jsx [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 import "./AddFirstTask.css";
+import { deleteLocalStorage } from "../../utils/localStorage";
 function AddFirstTask() {
+  const { modal, setModal } = useModal();
   return (
     <div className="first-task-wrapper display-flex flex-col justify-content-center align-items-center gap-1">
       <h2 className="txt-white m-0">Add Task</h2>
-      <button className="txt-white txt-lg btn-task-large">
+      <button
+        onClick={() => {
```
Comment (rehanmohiuddin) [inline src/components/AddFirstTask/AddFirstTask.jsx]: instead of inline function add handler to it

File: src/components/Modal/Modal.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  text-transform: capitalize;
+}
+.task-input::placeholder {
+  color: var(--bb-color-gray);
+  opacity: 0.6;
+  font-style: italic;
+  text-transform: initial;
+}
+.task-description {
+  font-size: 1.3rem;
```
Comment (rehanmohiuddin) [inline src/components/Modal/Modal.css:35]: add some standard css instead of hard coding

**Review by rehanmohiuddin — APPROVED** [review]
Liked The Use Of Constants in actions

**Pull request merged**